src/modules/navigator/mission_block.cpp

```diff
@@ -322,12 +326,18 @@ MissionBlock::mission_item_to_position_setpoint(const struct mission_item_s *ite
 
 	case NAV_CMD_LAND:
 		sp->type = position_setpoint_s::SETPOINT_TYPE_LAND;
+		if(_navigator->get_vstatus()->is_vtol && _param_vtol_wv_land.get()){
```

Comment:
We should add this to is_reached() in the mission if the command is a normal waypoint and a 1-2 second timeout occurred.